DTS-STN/Service-Canada-Labs
# Update Home Page

# Description

This PR is to [update the homepage](https://dev.azure.com/VP-BD/DECD/_boards/board/t/Service%20Canada%20Labs/Stories%20and%20Activities/?workitem=102618) to match the new design.

## Acceptance Criteria

Homepage should match the design seen [here](https://www.figma.com/file/ke3SrZNGMP3AjwzMMuEgbo/%5Bcurrent%5D-BDM-DECD-SC-Labs?node-id=11385%3A29412&t=tuHsqD2nEtypPqi8-0). See that it is responsive across viewport sizes.

## Test Instructions

1. Open home page
2. See that it matches new design

## Checklist
